tests/desktop/consumer_pages/test_home_page.py

```diff
@@ -60,7 +60,7 @@ def test_opening_category_pages_from_categories_menu(self, mozwebqa):
             Assert.true(category_page.popular_tab_class == 'active')
 
             # only check the first three apps in the category
-            for a in range(3):
+            for a in range(min(len(apps), 3)):
```

Comment:
With the stability branch merged I don't think we need this any more. We do `for app in apps[:3]:`, which will return up to the first three apps in the list.